yb-voyager/cmd/exportSchema.go

```diff
@@ -245,7 +246,7 @@ func applyMigrationAssessmentRecommendations() error {
 	return nil
 }
 
-func applyShardedTablesRecommendation(shardedTables []string) error {
+func applyShardedTablesRecommendation(shardedTables []string) (err error) {
```

Comment:
why give the return type a name? I generally feel like this approach is generally error prone. (especially when it comes to defer funcs where we might want to ignore the error)

Reply:
hmm i give it for specific use cases. Like in this case, i have a common variable defined `modifiedSqlStmt` used under multiple if-else conditions,like this:
```
if ....
         modifiedSqlStmt, err = func1()
else ...
         modifiedSqlStmt, err = func2()
```

so i would have to define err variable before hand. I could do when required - `var err error`

But for the defer case you mentioned, i agree its needs to be taken care of but that anyways we should use a separate variable in that case than the normal errors one.